pillar.example

```diff
@@ -37,15 +37,17 @@ salt:
     salt_ssh: 'salt-ssh'
     pyinotify: 'python-pyinotify'  # the package to be installed for pyinotify
 
-  # Set which salt repository to use, default to https://repo.saltproject.io
-  # For older releases use https://archive.repo.saltproject.io
-  repo: 'https://archive.repo.saltproject.io'
+  # Set which salt repository to use, default to https://packages.broadcom.com/artifactory
+  # yamllint disable rule:line-length
+  repo: 'https://packages.broadcom.com/artifactory'
+  repo_key_url: 'https://packages.broadcom.com/artifactory/api/security/keypair/SaltProjectKey/public'
```

Comment:
I actually put that in the wrong place on the first review. the disable rule needed to be before the comment on line 40. 

```suggestion
  # yamllint disable rule:line-length
  # Set which salt repository to use, default to https://packages.broadcom.com/artifactory
  repo: 'https://packages.broadcom.com/artifactory'
  repo_key_url: 'https://packages.broadcom.com/artifactory/api/security/keypair/SaltProjectKey/public'
```

Reply:
Done.  I cut the line